### PR TITLE
fix(swap): Show slippage warning when minReceived is 0

### PR DESCRIPTION
**Pull request by michalszorad**
resolves https://emurgo.atlassian.net/browse/YOMO-946

To show the warning:

Create Limit order 1 ADA for 1 MILK at price 1 and slippage 1%

Because slippage is > 0 and MILK does not have decimals, you can get 0 MILK

Screenshot 2023-11-01 at 15 26 48

### CODE REVIEW

**Comment by michalszorad** [comment]
Waiting for a confirmation from MuesliSwap team to confirm whether buy amount = buy amount - slippage

**Comment by michalszorad** [comment]
Waiting for a confirmation from product

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/SlippageWarning/SlippageWarning.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {Button, Spacer, useModal} from '../../../../../../components'
+import {useStrings} from '../../../../common/strings'
+
+export interface Props {
+  onClose?: () => void
+  onSubmit?: () => void
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/SlippageWarning/SlippageWarning.tsx]:
```suggestion
  onSubmit: () => void
```

File: apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/SlippageWarning/SlippageWarning.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+storiesOf('Swap Slippage Tolerance Warning', module).add('Initial', () => <Initial />)
+
+const Initial = () => {
+  const {openModal, content, closeModal} = useModal()
+  const strings = useStrings()
+  return (
+    <SelectedWalletProvider wallet={walletMocks.wallet}>
+      <SwapProvider swapManager={mockSwapManager}>
+        <SwapFormProvider>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/SlippageWarning/SlippageWarning.stories.tsx]: decorator?

**Review by stackchain — APPROVED** [review]

**Comment by michalszorad** [comment]
Got confirmation `buyAmountWIthSlipppage` is correct to be used

**Comment by michalszorad** [comment]
**Warning with a ticket and slippage**
Screenshot 2023-11-01 at 16 55 31




**Disabled confirm button on confirm tx screen**
Screenshot 2023-11-01 at 16 55 41

**Review by stackchain — APPROVED** [review]

**Pull request merged**